src/bionics.h

```diff
@@ -80,6 +80,8 @@ struct bionic_data {
     float weight_capacity_modifier;
     /**Bonus to weight capacity*/
     units::mass weight_capacity_bonus;
+    /**Stat bonus*/
+    std::map<std::string, int> stat_bonus;
```

Comment:
Would be better to use an enumeration as key here. Currently one can add a JSON entry with "dexterity" as key 
```JSON
"stat_bonus": [ [ "dexterity", 2 ] ],
```
And the game will load it without complaining, but it will have no effect on the game at all, it's effectively ignored.

The game should load the string, convert it an enumeration value and *complain* if the string does not match one of the known stats, so the author of the JSON data can fix that.